nifi-docker/dockerhub/sh/start.sh

```diff
@@ -79,11 +79,12 @@ case ${AUTH} in
 esac
 
 # Continuously provide logs so that 'docker logs' can    produce them
-tail -F "${NIFI_HOME}/logs/nifi-app.log" &
 "${NIFI_HOME}/bin/nifi.sh" run &
 nifi_pid="$!"
+tail -F --pid=${nifi_pid} "${NIFI_HOME}/logs/nifi-app.log" &
 
-trap "echo Received trapped signal, beginning shutdown...;" KILL TERM HUP INT EXIT;
+trap 'echo Received trapped signal, beginning shutdown...;./bin/nifi.sh stop;exit 0;' KILL TERM HUP INT;
```

Comment:
I know it is not added in this change, but let's remove `KILL` SIGNAL_SPEC, as trap cannot / nothing can catch sigkill.

Reply:
Sorry for the late response... we're on the eve of a COVID-19 curfew here...
Yes, I agree with you KILL is superfluous.